gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -162,6 +162,10 @@ message InterceptHtlcResponse {
 
   // The index of the incoming htlc in the incoming channel
   uint64 htlc_id = 5;
+
+  // The HTLC payment hash.
+  // Value is not guaranteed to be unique per intercepted HTLC
```

Comment:
This comment confused me a little initially, since its phrased in the negative. I think something like "HTLCs corresponding to the same payment will have the same payment hash" is more clear.

Reply:
Done.